src/features/planning/MatrixQuadrant.tsx

```
@@ -0,0 +1,67 @@
+						<p className="text-sm text-white/40">Drop tasks here</p>
+					</div>
+				) : (
+					<div className="space-y-2">
+						{tasks.map((task) => (
+							<div key={task.id} className="relative group">
+								<TaskCard task={task} />
+								<button
+									onClick={() => onReturnTask(task.id)}
+									className="absolute -left-2 top-1/2 -translate-y-1/2 opacity-0 group-hover:opacity-100 transition-opacity p-1 bg-retro-medium rounded-full hover:bg-retro-light"
+									title="Return to sidebar"
+								>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)